src/ADNLPProblems/hs10.jl

```diff
@@ -3,7 +3,7 @@ export hs10
 function hs10(; n::Int = default_nvar, type::Val{T} = Val(Float64), kwargs...) where {T}
   x0 = T[-10; 10]
   f(x) = x[1] - x[2]
-  c(x) = T[-3 * x[1]^2 + 2 * x[1] * x[2] - x[2]^2 + 1]
+  c(x) = [-3 * x[1]^2 + 2 * x[1] * x[2] - x[2]^2 + 1]
   lcon = T[0.0]
```

Comment:
explicit Float64 here

Reply:
There is a unit test for this and the type seems correct (not restricted to Float64).